pocs/utils/images/__init__.py

```diff
@@ -201,7 +199,14 @@ def _make_pretty_from_cr2(fname, timeout=15, **kwargs):  # pragma: no cover
     return fname.replace('cr2', 'jpg')
 
 
-def create_timelapse(directory, fn_out=None, file_type='jpg', **kwargs):
+def create_timelapse(
+        directory,
+        fn_out=None,
+        file_type='jpg',
+        overwrite=False,
+        timeout=60,
+        verbose=False,
+        **kwargs):
     """Create a timelapse
 
     A timelapse is created from all the jpg images in a given `directory`
```

Comment:
jpg is the default type of file, but we allow for others. What other types would ffmpeg support? Probably gif and png. Speaking of which, I've wondered about using png in place of jpg to avoid lossy artifacts.

Reply:
jpgs come directly out of the CR2 without us having to create/convert anything so it's the easiest. Would be open to another solution if practical.

Reply:
Updated docs to not be jpg specific.